src/dia/ports.mjs

```diff
@@ -334,6 +334,26 @@ export const elementPortPrototype = {
         return this;
     },
 
+    /**
+     * @param {string|Port|number} before
+     * @param {object} port
+     * @param {object} [opt]
+     * @returns {joint.dia.Element}
+     */
+    insertPort: function(before, port, opt) {
+        const index = (typeof before === 'number') ? before : this.getPortIndex(before);
+
+        if (!util.isObject(port) || Array.isArray(port)) {
+            throw new Error('Element: addPort requires an object.');
```

Comment:
Could you please change this to `dia.Element: insertPort requires an object.`?

Reply:
Hi @kumilingus , oh, copy-paste error for sure :-( I'll change this ASAP, should've seen this myself. THanks for the review